### DISCUSSION TITLE
PnP Modern Search and SP Subscriptions (OnPrem)

### DISCUSSION THREAD

**Discussion by DRNRW** [microsoft-search/pnp-modern-search#1712] [Q&A] [answered]
Can PnP Modern Search also be used for SP Subscriptions?

**Comment by FranckyC** [comment, marked as answer]
If you are talking about SharePoint on-premise, no it is not supported.

**Reply by DRNRW** [reply]
Yes, I am talking about SharePoint Subscriptio OnPremise.
Thanks for the answer.
